examples/stable/test/end_to_end/candid_rpc/candid_backtraces/src/candid_backtraces.ts

```diff
@@ -0,0 +1,14 @@
+import { IDL, query, trap } from 'azle';
+
+export default class {
+    @query([IDL.Text], IDL.Bool)
```

Comment:
Use the never TypeScript type for the return type, and use IDL.Empty for the IDL return type

Reply:
Then get rid of the `return false`